Style.css

```
@@ -1,43 +1,69 @@
-@import url('https://fonts.googleapis.com/css2?family=PT+Serif&display=swap');
-body {
-  margin: 0;
-  font-size: 12px;
-  line-height: 1.6;
-  width: 1200px;
+/* pt-serif-regular - latin */
+@font-face {
+  font-family: 'PT Serif';
+  font-style: normal;
+  font-weight: 400;
+  src: url('../fonts/pt-serif-v12-latin-regular.eot'); /* IE9 Compat Modes */
+  src: local(''),
+       url('../fonts/pt-serif-v12-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../fonts/pt-serif-v12-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../fonts/pt-serif-v12-latin-regular.woff') format('woff'), /* Modern Browsers */
+       url('../fonts/pt-serif-v12-latin-regular.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('../fonts/pt-serif-v12-latin-regular.svg#PTSerif') format('svg'); /* Legacy iOS */
 }
-*,
- *::before,
- *::after {
-  box-sizing: border-box;
+/* pt-serif-700 - latin */
+@font-face {
+  font-family: 'PT Serif';
+  font-style: normal;
+  font-weight: 700;
+  src: url('../fonts/pt-serif-v12-latin-700.eot'); /* IE9 Compat Modes */
+  src: local(''),
+       url('../fonts/pt-serif-v12-latin-700.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+       url('../fonts/pt-serif-v12-latin-700.woff2') format('woff2'), /* Super Modern Browsers */
+       url('../fonts/pt-serif-v12-latin-700.woff') format('woff'), /* Modern Browsers */
+       url('../fonts/pt-serif-v12-latin-700.ttf') format('truetype'), /* Safari, Android, iOS */
+       url('../fonts/pt-serif-v12-latin-700.svg#PTSerif') format('svg'); /* Legacy iOS */
 }
 
-.clearfix ::after {
+body {
+  margin: auto;
+}
+.clearfix::after { 
   content: '';
-  clear: 
-  display: block;
+  clear: both; 
+  display: inline;
 }
 
-.logotype {
+.container {
+  max-width: 100%;
+  margin-right: 0 auto;
+  margin-left: 0 aut;
+}
+.header-logo {
   font-size: 12px;
-  font-weight: 400;
+  font-weight: 700;
+  font-family: PT Serif;
```

always add a generic default name

https://developer.mozilla.org/ru/docs/Web/CSS/font-family#%D0%97%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%B8%D1%8F
```suggestion
  font-family: PT Serif, serif;
```